src/NuGet.Core/NuGet.Protocol/PackagesFolder/LocalPackageInfo.cs

```diff
@@ -43,9 +47,52 @@ public class LocalPackageInfo
         /// </summary>
         public NuspecReader Nuspec => _nuspec.Value;
 
+        /// <summary>
+        /// Package files with OPC files filtered out.
+        /// Cached to avoid reading the same files multiple times.
+        /// </summary>
+        public IReadOnlyList<string> Files => _files.Value;
+
         public override string ToString()
         {
             return Id + " " + Version + " (" + (ManifestPath ?? ZipPath) + ")";
```

Comment:
i know this is old code, and if it is called pretty often, am wondering if it makes sense to use stringbuilder to append strings here?